azure-pipelines.yml

```
@@ -64,7 +64,7 @@ jobs:
       set -e
       brew update
       (cd $(brew --repository)/Library/Taps/homebrew/homebrew-core/ && git fetch --shallow-since=${HBCORE_DATE} && git checkout ${HBCORE_REF})
-      brew install gnu-time opam pkg-config gtksourceview3 adwaita-icon-theme
+      brew install gnu-time opam pkg-config gtksourceview3 adwaita-icon-theme || true
```

If you're going to ignore the failure, why not just remove the failing package?
```suggestion
      brew install gnu-time opam pkg-config gtksourceview3 # adwaita-icon-theme
```

---

Is this the failing package? From the discussion on Zulip, this was not my understanding.